chargeOclock/Source/Models/Client.swift

```diff
@@ -14,8 +14,8 @@ protocol ClientSettable {
 
 class Client {
 
-	var id: Int = 0
-	var isDirty: Bool = false
+	var id: Int? = nil
+	var isDirty: Bool!
```

Comment:
It's implicitly unwrapped to make sure that anyone who makes changes to any initialiser *must* also take care (and initialise) the `isDirty` flag. That's my reasoning. But I'm not standing strong on this reason, as I can see benefits for the other way and could easily leave the default value set to `false`.
Do you prefer the implementation with the default value for the `isDirty` parameter? Why?